src/components/elements/lockup/lockup.tsx

```diff
@@ -31,7 +32,7 @@ export type LockupProps = {
 }
 type ImageFormats = "png" | "svg+xml" | "jpeg" | "zip"
 
-export const Lockup = ({lockupOption = "a"}) => {
+export const Lockup = ({lockupOption = "h"}) => {
```

Comment:
@pookmish on this line the lockup option is set.  If the select is further down in the actual code, how can it fill in this option? Would it have to `{lockupOption}` from the select but wouldn't that be an error?